pkg/kubelet/kubelet_getters.go

```diff
@@ -274,6 +274,24 @@ func (kl *Kubelet) getPodVolumePathListFromDisk(podUID types.UID) ([]string, err
 	return volumes, nil
 }
 
+func (kl *Kubelet) getMountedVolumePathListFromDisk(podUID types.UID) ([]string, error) {
```

Comment:
this function is not called

Reply:
Yeah, not sure why this function was introduced and then got commented out in code below.

Reply:
sorry for the confusion. It is being called. I was testing at that time.